### PR TITLE
Seqsampling

### PR DESCRIPTION
**Pull request by sylvainczx**
Sequential sampling procedures to compute confidence intervals.

### CODE REVIEW

File: doc/src/confidence_intervals.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. _MMW confidence interval:
+.. Confidence intervals:
```
Comment (DLWoodruff) [inline doc/src/confidence_intervals.rst:1]: Anchors need a leading underscore

File: mpisppy/confidence_intervals/seqsampling.py [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+    solvername: str, optional
+        Name of the solver used. The default is "gurobi"
+    solver_options: dict, optional
+        Solving options. The default is None.
+    crops_multiplier: int, optional
+        A parameter of the farmer model. The default is 1.
+
+    Returns
+    -------
+    path: str
```
Comment (DLWoodruff) [inline mpisppy/confidence_intervals/seqsampling.py:49]: Probably, this function should return xhat and not write it to a file

File: mpisppy/utils/xhat_eval.py [hunk 3]
```diff
@@ -57,7 +57,8 @@ def __init__(
         #TODO: CHANGE THIS AFTER UPDATE
         self.PH_extensions = None
         
-    
+        self.subproblem_creation(self.verbose)
+        self._create_solvers()
```
Comment (bknueven) [inline mpisppy/utils/xhat_eval.py:61]: The classes derived from `Xhat_Eval` probably also do this somewhere. We should eliminate the associated calls in the derived classes to ensure we're not doing extra work.

**Review by bknueven — CHANGES_REQUESTED** [review]
Need to fix issue with Xhat_Eval spoke friend classes

**Review by bknueven — APPROVED** [review]
I fixed the issue I had

**Pull request merged**